Service/Request/RequestFactory.php

```diff
@@ -182,6 +182,7 @@ private function parseQueryParams(string $path): array
 
         if (null !== $queryParamsString) {
             parse_str($queryParamsString, $queryParamsArray);
+            $queryParamsArray = $this->normalizeQueryParams($queryParamsArray);
```

Comment:
Could we have a test for this?

Reply:
It's possible to add test, but to add it will increase the complexity in order to extract the function to it's own class, or global function to properly generate the  test case for it.
Other option could be to use reflection on the test to change the visibility of the function and test it in that way.
What option do you think is more suitable for this ?